composer.json

```diff
@@ -28,23 +28,23 @@
         "php": ">=7.0.0 <7.3",
         "helhum/typo3-console-plugin": "^2.0.2",
 
-        "typo3/cms-backend": "^8.7.10 || ~9.1.0",
-        "typo3/cms-core": "^8.7.10 || ~9.1.0",
-        "typo3/cms-extbase": "^8.7.10 || ~9.1.0",
-        "typo3/cms-extensionmanager": "^8.7.10 || ~9.1.0",
-        "typo3/cms-fluid": "^8.7.10 || ~9.1.0",
-        "typo3/cms-frontend": "^8.7.10 || ~9.1.0",
-        "typo3/cms-install": "^8.7.10 || ~9.1.0",
-        "typo3/cms-saltedpasswords": "^8.7.10 || ~9.1.0",
-        "typo3/cms-scheduler": "^8.7.10 || ~9.1.0",
+        "typo3/cms-backend": "^8.7.10 || ~9.1.0 || ~9.2.0",
```

Comment:
Would it make sense to use `^9.0 < 9.3` here? This way the constraint won't grow each time a new release is made.

Reply:
First: We're not compatible with TYPO3 9.0 any more.
Second: I will remove 9.1 support with the next minor release anyway.

Plan is to only support the current sprint release and leave only older console versions for older sprint releases. Otherwise the compatibility layer would grow too much, which is a bigger issue than verbose constraints in composer.json

This one here is only to get a 9.2.0 compatible version out as quickly as possible as bugfix release.

Reply:
Thanks for explaining, makes sense.